test/dlc_tests/container_tests/bin/testTFKerasHVD.py

```diff
@@ -29,8 +29,8 @@
 if data_type == "AMP":
     # if this code block get excecuted, data type is mixed-precision AKA AMP. If not, it is FP32.
     # When ampere comes and TF32 is available, we need to extend the test to run TF32 as well
-    policy = tf.keras.mixed_precision.experimental.Policy('mixed_float16', 128)
-    tf.keras.mixed_precision.experimental.set_policy(policy)
+    policy = tf.keras.mixed_precision.Policy('mixed_float16')
+    tf.keras.mixed_precision.set_global_policy(policy)
```

Comment:
is this backward compatible with previous TF versions?

Reply:
Confirmed that this change is compatible till at least TF2.5